ui/ducks/send/send.js

```diff
@@ -45,7 +45,8 @@ import {
   getIsMultiLayerFeeNetwork,
   getEnsResolutionByAddress,
   getSelectedAccount,
-  getSelectedAddress,
+  getSelectedInternalAccount,
+  getSelectedInternalAccountWithBalance,
```

Comment:
is there no way to derive all the information we need from just one of these methods?

Reply:
Is that only for the benefit of calling on method instead of two in this file?

Reply:
it will save a re render since these methods are state calls and have side effects. That would be the only reason.

Reply:
Following that logic then we should only have one method that exposes the entire state, by the looks of it, if the balance is not necessary then the selector should not provide it. This is also not a component, it's a thunk action creator generator